modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/ModelController.java

```diff
@@ -41,29 +45,34 @@
     public Boolean getModel() {
         return true;
     }
+
+    @ApiOperation(value = "Get model info by providing metric key")
     @RequestMapping(value = "/api/model", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<ModelDto> getModel(@RequestParam("metricKey") String metricKey) {
         return modelService.getModels(metricKey);
     }
 
-    @RequestMapping(value = "/api/addModelParams", method = RequestMethod.POST)
-    public HttpStatus addModelParams(@RequestBody ModelParams modelParams) {
+    @ApiOperation(value = "Add model hyper params")
+    @RequestMapping(value = "/api/addModelParams", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> addModelParams(@RequestBody ModelParams modelParams) {
         modelService.addModelParams(modelParams);
-        return HttpStatus.OK;
+        return new ResponseEntity<String>("Model params saved successfully", HttpStatus.OK);
     }
 
-    @RequestMapping(value = "/api/markToRebuild", method = RequestMethod.PUT)
-    public HttpStatus markToRebuild(@RequestBody RebuildParams rebuildParams) {
+    @ApiOperation(value = "Mark model to rebuild")
+    @RequestMapping(value = "/api/markToRebuild", method = RequestMethod.PUT, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> markToRebuild(@RequestBody RebuildParams rebuildParams) {
         modelService.markToRebuild(rebuildParams.getModelUUID(), rebuildParams.getMetricKey(),
                 rebuildParams.getToRebuild());
-        return HttpStatus.OK;
+        return new ResponseEntity<String>("Model marked for rebuild", HttpStatus.OK);
     }
```

Comment:
Simpler: return a String, and just return "Model marked for rebuild" as the string. Spring will automatically 200 the response. Same for the other cases. Keep the code tight here.